action.yml

```diff
@@ -102,13 +103,13 @@ runs:
         retention-days: 5
 
     - name: Validate hyperlinks
+      if: ${{ inputs.VALIDATE_LINKS == "true" }}
```

Comment:
As `fromJson(steps.prepare.outputs.validate).links` processes raw inputs, we should use that instead of `inputs` directly.

Reply:
OK, I think I understood what was happening for those outputs, and updated the condition